_FIPS201/frontend.md

```diff
@@ -635,8 +635,8 @@ management requirements are also specified.
 #### 4.2.2.1 PIV Authentication Key {#s-4-2-2-1}
 This key **SHALL** be generated on the PIV Card. The PIV Card **SHALL NOT**
 permit exportation of the PIV authentication key. The cryptographic operations that use the PIV
-authentication key **SHALL** be available only through the contact and virtual contact interfaces of the
-PIV Card. Private key operations **MAY** be performed using an activated PIV Card without explicit
+authentication key **SHALL** be available through the contact and virtual contact interfaces of the
```

Comment:
Being somewhat pedantic, should this be "SHALL" or should it instead be "MAY only"?

Or, perhaps more accurately, "SHALL be available over the contact interface and MAY be available over the virtual contact interface".  You wouldn't want a card that can't do PIV auth over the contact interface, but you can have one that doesn't have a VCI.

Reply:
```suggestion
authentication key **SHALL** be available through the contact interface and **MAY** additionally be available over the virtual contact interface of the
```